src/v2/guide/components-registration.md

```diff
@@ -122,20 +122,20 @@ export default {
 }
 ```
 
-Note that in ES2015+, placing a variable name like `ComponentA` inside an object is shorthand for `ComponentA: ComponentA`, meaning the name of the variable is both:
+Note que en ES2015+, colocar un nombre de variable como `ComponentA` dentro de un objeto es la abreviatura de `ComponentA: ComponentA`, lo que significa que el nombre de la variable es ambos:
 
-- the custom element name to use in the template, and
-- the name of the variable containing the component options
+- el nombre del elemento personalizado para usar en la plantilla, y
+- el nombre de la variable que contiene las opciones de componentes
 
-## Module Systems
+## Sistemas de módulos
 
-If you're not using a module system with `import`/`require`, you can probably skip this section for now. If you are, we have some special instructions and tips just for you.
+Si no estás usando un sistema de módulos con `importar` o `requerir`, probablemente puedes saltarte esta sección por ahora. Si lo estás, tenemos algunas instrucciones y consejos especiales sólo para ti.
 
-### Local Registration in a Module System
+### Registro local en un Sistema de Módulos
 
-If you're still here, then it's likely you're using a module system, such as with Babel and Webpack. In these cases, we recommend creating a `components` directory, with each component in its own file.
+Si todavía estás aquí, es probable que estés usando un sistema de módulos, como con Babel y Webpack. En estos casos, recomendamos crear un directorio de `components`, con cada componente en su propio archivo.
```

Comment:
Same here:
```suggestion
Si todavía está aquí, es probable que estée usando un sistema de módulos, tales como Babel y Webpack. En estos casos, recomendamos crear un directorio de `components`, con cada componente en su propio archivo.
```

Reply:
> Same here

@Tintef  I think 'estée' should be 'esté'
'tales como ' is the right form.

Reply:
Yeah, my typo there

Reply:
> Yeah, my typo there

Hey @Tintef would you remove the 'e' so I can commit the suggestion or if you prefer I make all the changes as you suggested

Reply:
@PabloRN Edited, sorry for the late response